src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -206,7 +206,7 @@ export default [
         onPress: (closePopover, {reportAction, reportID}) => {
             Environment.getEnvironmentURL().then((environmentURL) => {
                 const reportActionID = parseInt(lodashGet(reportAction, 'reportActionID'), 10);
-                Clipboard.setString(`${environmentURL}/r/${reportID}/${reportActionID}`);
+                Clipboard.setString(`${environmentURL}r/${reportID}/${reportActionID}`);
```

Comment:
I don't think this change is needed if we rollback the commit for environment URL?